ggshield/cmd/sca/scan.py

```diff
@@ -45,8 +52,84 @@ def scan_all_cmd(
     ctx: click.Context,
     directory: Optional[Path],
     **kwargs: Any,
-) -> int:
+):
     """
     Scan a directory for SCA vulnerabilities.
     """
-    return 0
+    if directory is None:
+        directory = Path().resolve()
+
+    result = sca_scan_all(ctx, directory)
+    return result
+
+
+def sca_scan_all(ctx: click.Context, directory: Path):
```

Comment:
Maybe this should also be typed

Reply:
Indeed, but the actual return type is defined in another MR.